drivers/sensor/bmp388/Kconfig

```diff
@@ -12,6 +12,20 @@ menuconfig BMP388
 
 if BMP388
 
+config BMP388_READ_WHEN_SUSPENDED
```

Comment:
Move this to a devicetree property

Reply:
Hello Maureen,

The code in this PR does not change the description of the BMP388 hardware or affect its boot time configuration; it simply selects one of two possible behaviours when a read is attempted while the sensor is sleeping, so I think this should stay a KConfig option.

Maybe I am misunderstanding something;  I based my decision on the Zephyr documentation about [dt vs kconfig](https://docs.zephyrproject.org/latest/build/dts/dt-vs-kconfig.html):

> As another example, consider a device with a 2.4GHz, multi-protocol radio supporting both the Bluetooth Low Energy and 802.15.4 wireless technologies.
> 
> Devicetree should be used to describe the presence of the radio hardware, what driver or drivers it’s compatible with, etc.
> 
> Boot-time configuration for the radio, such as TX power in dBm, should also be specified using devicetree.
> 
> Kconfig should determine which software features should be built for the radio, such as selecting a BLE or 802.15.4 protocol stack.